### PR TITLE
Delete selected layers

### PR DESCRIPTION
**Pull request by TrueDoctor**
#139

---
This change is [Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/173)

### CODE REVIEW

File: client/web/src/components/panels/LayerTree.vue [hunk 1]
```diff
@@ -68,6 +74,9 @@
 			margin-left: 4px;
 			padding-left: 16px;
 		}
+		.selected {
+			background: var(--color-accent);
```
Comment (Keavon) [inline client/web/src/components/panels/LayerTree.vue:78]: Please also add a line with `color: var(--color-f-white);`

File: core/editor/src/document/document_message_handler.rs [hunk 2]
```diff
@@ -95,11 +96,24 @@ impl MessageHandler<DocumentMessage, ()> for DocumentMessageHandler {
 				self.active_document_mut().layer_data(&path).expanded ^= true;
 				responses.extend(self.handle_folder_changed(path));
 			}
+			DeleteSelectedLayers => {
+				// TODO: Replace with drain_filter https://github.com/rust-lang/rust/issues/59618
+				log::debug!("{:?}", self.active_document().layer_data);
+				let paths: Vec<Vec<LayerId>> = self.active_document().layer_data.iter().filter_map(|(path, data)| data.selected.then(|| path.clone())).collect();
```
Comment (Keavon) [inline core/editor/src/document/document_message_handler.rs:101]: Do we have a type alias for `Vec<LayerId>` or `Vec<Vec<LayerId>>`? If we do have one let's not forget to use it here.

**Review by TrueDoctor — COMMENTED** [review]
We don't

> *[Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/173#-:-MbmGYaL04VDLTkHbMYJ:b-5smf3x)* status: 0 of 7 files reviewed, all discussions resolved

File: core/editor/src/document/document_message_handler.rs (hunk 2 above)
Comment (TrueDoctor) [reply core/editor/src/document/document_message_handler.rs:109]: We don't

**Review by Keavon — APPROVED** [review]

**Review by Keavon — APPROVED** [review]
> Reviewed 1 of 7 files at r2, 1 of 3 files at r3, 1 of 2 files at r4, 9 of 9 files at r5.
> *[Reviewable](https://reviewable.io/reviews/graphiteeditor/graphite/173)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @TrueDoctor)

**Pull request merged**